listenbrainz/db/stats.py

```diff
@@ -52,11 +53,10 @@ def insert_user_stats(user_id, artists, recordings, releases, artist_count):
              artist_count (int): the total number of artists listened to by the user
     """
 
+    artist_range = artists['range']
+    del artists['range']
     artist_stats = {
-        'count': artist_count,
```

Comment:
this count is used in the user page, if we're removing this, we should also fix that page. I'm ok with keeping it too.